source/elm-live.js

```diff
@@ -52,8 +52,52 @@ module.exports = (argv, options) => {
     return SUCCESS;
   }
 
+  const auxiliaryBuild = (execPath) => {
+    if (!execPath) {
+      return { fatal: false, exitCode: SUCCESS };
+    }
+
+    const process = spawnSync(execPath, [], {
+      stdio: [inputStream, outputStream, outputStream],
+    });
+
+    if (process.error && process.error.code === 'ENOENT') {
+      outputStream.write(
+`\n${dim('elm-live:')}
+  I can’t find the command ${bold(execPath)}!
+  Please make sure you can call ${bold(execPath)}
+  from your command line.
+`
```

Comment:
There should probably be a blank line after the block of text. Just to separate this stuff from any following output.

If you see anywhere else where we’ve forgotten the blank line, feel free to [boy scout](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule) 🙂 ⛺️